ocp4/cpe/ocp4-cpe-dictionary.xml

```diff
@@ -7,4 +7,34 @@
             <!-- the check references an OVAL file that contains an inventory definition -->
             <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="filename">installed_app_is_ocp4</check>
       </cpe-item>
+      <cpe-item name="cpe:/o:redhat:enterprise_linux:8">
```

Comment:
I think cpe (RHEL8) is not needed here. At least from my tests it didn't require to be here.

Reply:
Fixed